Aleksandra_Pyvovar/js/scripts.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Created by John Doe on 17.10.2017.
+ */
+document.onreadystatechange = function () {
+  if (document.readyState === 'interactive') {
+    initCalculator();
+  }
+};
+
+function initCalculator() {
+  var
+    calculator = document.getElementById('calculator'),
+    first = document.getElementById('first-number'),
+    second = document.getElementById('second-number'),
+    action = document.getElementById('action');
+
+
+  calculator.onsubmit = function(event) {
+    var
+      firstNumber = parseFloat(first.value),
+      secondNumber = parseFloat(second.value),
+      actionType = action.value.trim(),
+      result = calculate(firstNumber, secondNumber, actionType);
+
+    event.preventDefault();
+
+    if ((result !== undefined) && (!isNaN(result))  ) {
+      console.log('hello');
+      resultRender(firstNumber, secondNumber, actionType, result);
+      this.reset();
+    }
+  }
+}
+
+function calculate(firstNumber, secondNumber, action) {
+  var result;
+
+  switch(action) {
```

Comment:
Don't repeat yourself, You can write as follows:
```javascript
case '+':
case 'add':
  result = firstNumber + secondNumber;
  break;
...
```

Reply:
Yes, I've seen it in Madfin's code)

Reply:
Please, refactor Your solution.

Reply:
Great job!
But why You put all in one line - it's hard to read.
And because of difference between length of synonym words - **`result = ...`** each time not in the same place.
If more then one action needed, like 3 or 5 - will You also write code in one line?